flottform/where-are-you-at/src/routes/+page.svelte

```diff
@@ -64,9 +63,30 @@
 			$currentState = 'error';
 		});
 	});
+	const copyLinkToClipboard = () => {
+		const flottformLink = (document.querySelector('.link-offer') as HTMLDivElement).innerText;
+		const copyToClipboardButton = document.querySelector(
+			'.copy-button-link-wrapper .copy-to-clipboard'
+		) as HTMLButtonElement;
+
+		navigator.clipboard
+			.writeText(flottformLink)
+			.then(() => {
+				copyToClipboardButton.innerText = '✅';
+				setTimeout(() => {
+					copyToClipboardButton.innerText = '📋';
+				}, 1000);
+			})
+			.catch((error) => {
+				copyToClipboardButton.innerText = `❌ Failed to copy: ${error}`;
+				setTimeout(() => {
+					copyToClipboardButton.innerText = '📋';
+				}, 1000);
+			});
+	};
 </script>
 
-<div class="w-full min-h-svh grid place-items-center">
+<div class="w-full grow grid place-items-center">
```

Comment:
I would also add some padding here so it has some space on smaller screens
![image](https://github.com/user-attachments/assets/867b2528-f94f-43f9-9360-d247498c1ecc)
```suggestion
<div class="w-full grow grid place-items-center px-4">
```